lms/djangoapps/instructor_task/tasks_helper/misc.py

```diff
@@ -153,7 +154,9 @@ def cohort_students_and_upload(_xmodule_instance_args, _entry_id, course_id, tas
                 cohorts_status[cohort_name] = {
                     'Cohort Name': cohort_name,
                     'Students Added': 0,
-                    'Students Not Found': set()
+                    'Students Not Found': set(),
+                    'Invalid Email Addresses': set(),
+                    'Preregistered Users': 0
```

Comment:
Also, my understanding from talking to @sstack22 is that we want to display the email addresses of preregistered learners, not just a count of the number. It's in the AC for the story.